bin/root-transition.sh

```diff
@@ -151,19 +154,67 @@ transition() {
   echo "Please save them safely and privately."
 }
 
+check_version_namespace() {
+  OUT=`kubectl get po -n $1 -o yaml | grep "istio\/proxyv2\:1\."`
+
+  for LINE in $OUT;
+  do
+    if [[ $LINE == *"istio/proxyv2"* ]]; then
+      LINE=${LINE#"gke.gcr.io/istio/proxyv2:"};
+      LINE=${LINE#"docker.io/istio/proxyv2:"};
+      LINE=${LINE#"istio/proxyv2:"};
+      VER=${LINE%%"-gke.0"};
+      echo "Istio proxy version: $VER";
+    fi
+  done
+}
+
+check_version() {
+  # shellcheck disable=SC2006
+  NS=`kubectl get ns | grep -v "STATUS" | grep -v "kube-system" | grep -v "kube-public" | awk '{print $1}'`
+
+  for n in $NS
+  do
+    echo "Checking namespace: $n"
+    # shellcheck disable=SC2086
+    check_version_namespace $n
```

Comment:
Why can't you use quotes?

Also, a better pattern for variables is `${variable}` instead of `$variable`

Reply:
Done. Sorry don't have much experience on bash. Trying my best.